README.md

```diff
@@ -1,31 +1,37 @@
+---
+skip_index: true
```

Comment:
I _think_ this may be unnecessary since I don't think this file is ever included into the site that is using the theme gem.